frontend/src/components/workbench/panel/workflow/form.tsx

```diff
@@ -333,17 +331,15 @@ function WorkflowReturnValueTooltip() {
     <div className="flex w-full flex-col space-y-4">
       <div className="flex w-full items-center justify-between text-muted-foreground">
         <span className="font-mono text-sm font-semibold">
-          Return Value Expression
+          Output Schema
         </span>
         <span className="text-xs text-muted-foreground/80">(optional)</span>
       </div>
       <span className="w-full text-muted-foreground">
-        Define an expression that will be evaluated and returned as the
-        workflow&apos;s output.
+        Key-value pairs that define the data returned by the workflow run.
```

Comment:
> Key-value pairs

This would make it seem like it can only return JSON objects, but there may be certain use cases that you'd like to return a list or just a simple value so perhaps this should be reworded

Reply:
Yes. But this doesn't have to be an expression right?

Reply:
Updated